New package: CTFlows v0.4.5

JuliaRegistrator opened the pull request:
- Registering package: CTFlows
- Repository: https://github.com/control-toolbox/CTFlows.jl
- Created by: @ocots
- Version: v0.4.5
- Commit: 6534a9100043fd32a7c81dc7382e2b18c64e9833
- Reviewed by: @ocots
- Reference: https://github.com/control-toolbox/CTFlows.jl/issues/27#issuecomment-2212400105
- Description: Flows: classical, Hamiltonian, from OCP and more
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a835982e8e352f9aa8e03ac29f9bce55cf585cc4eaa6d9dd348a04faa8fd2d6f00d665a84e9f4f0c1efbcd840fdd605bde2778f1917ad1576acc230a1b1ad7bd2c82ba53533084ea8801f82948ecc5b00f186b9b27aa0658efee2c7b8f33e692a7d560da09ec3f159e2737122d1129757a30fd5a0a288e5d2c8e1ae47b7a2e484618fd -->

ocots commented:
[noblock] Dear Admin,

Mainly with @jbcaillau, @PierreMartinon and @joseph-gergaud, we develop an optimal control toolbox :

- [GitHub repo](https://github.com/control-toolbox)
- [Documentation](https://control-toolbox.org/docs/optimalcontrol/stable/)

This toolbox is composed of several packages:

```mermaid
flowchart TD
O(<a href='https://control-toolbox.org/docs/optimalcontrol/stable/'>OptimalControl</a>) --> B(<a href='https://control-toolbox.org/docs/ctbase/stable/'>CTBase</a>)
O --> D(<a href='https://control-toolbox.org/docs/ctdirect/stable/'>CTDirect</a>)
O --> F(<a href='https://control-toolbox.org/docs/ctflows/stable/'>CTFlows</a>)
F --> B
D --> B
```

We intend if accepted to switch from our [local registry](https://github.com/control-toolbox/ct-registry) to the general. We should move `CTBase.jl`, `CTDirect.jl`, `CTFlows.jl` and the main package `OptimalControl.jl`.

Note that we have presented it at the last JuliaCon: see the [presentation](https://control-toolbox.org/docs/optimalcontrol/v0.8/juliacon2023.html).

JuliaRegistrator committed:
New package: CTFlows v0.4.5

UUID: 1c39547c-7794-42f7-af83-d98194f657c2
Repo: https://github.com/control-toolbox/CTFlows.jl.git
Tree: 573f109d573fcd3cf0ebcb8ad4054be3d97e0815

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
